tests/client.js

```diff
@@ -19,13 +19,13 @@ Tinytest.add("ShareIt - {{> shareit}} template doesn't throw an exception from t
 
 
 Tinytest.addAsync("ShareIt - {{> shareit}} template renders valid share links", function(test, done) {
-  ShareIt.settings.useFB = true;
+  ShareIt.settings.sites.push('facebook');
   div = renderToDiv(Template.test_helper_shareit, dataContext)
   // This feels like a punt, should investigate more
   Meteor.setTimeout(function() {
     html = div.innerHTML
-    test.include(html, 'https://www.facebook.com/sharer/sharer.php?s=100&amp;p[url]=http%3A%2F%2Flocalhost%3A3000%2F&amp;p[title]=Test%20title&amp;p[summary]=undefine')
-    test.include(html, 'https://twitter.com/intent/tweet?url=http%3A%2F%2Flocalhost%3A3000%2F&amp;text=Test%20title')
+    test.include(html, 'https://www.facebook.com/sharer/sharer.php?s=100&amp;p[url]=http%3A%2F%2Flocalhost%3A10015%2Flocal&amp;p[title]=Test%20title&amp;p[summary]=undefined')
```

Comment:
Hmm, this needs to be not hardcoded, I think, or matched slimmer without the port on it somehow...

@samhatoum - any thoughts on a better watch to do string matching here with a url inside it?